lib/futurism/resolver/controller/renderer.rb

```diff
@@ -30,7 +34,7 @@ def setup_env!
             path = ActionDispatch::Journey::Router::Utils.normalize_path(uri.path)
             query_hash = Rack::Utils.parse_nested_query(uri.query)
 
-            path_params = Rails.application.routes.recognize_path(path)
+            path_params = recognize_path(url) # use full url to be more likely to match a url with subdomain constraints
```

Comment:
shouldn't we name that `recognize_url` then?

Reply:
Sure!